mock/Cargo.toml

```diff
@@ -28,6 +28,8 @@ env_logger = { version = "0.11" }
 hex = "0.4"
 miden-lib = { path = "../miden-lib" }
 miden-objects = { path = "../objects" , features = ["serde", "log", "testing"] }
+miden-prover = { package = "miden-prover", git = "https://github.com/0xPolygonMiden/miden-vm", branch = "next", default-features = false }
+miden-test-utils = { package = "miden-test-utils", git = "https://github.com/0xPolygonMiden/miden-vm", branch = "next", default-features = false }
```

Comment:
Using `miden-test-utils` here may cause issues because we've switched to using miden VM crates from crates.io - but `miden-test-utils` is not published to crates.io.

Looking quickly through the code, I couldn't see where we are using this dependency. Is it actually needed?

Reply:
changed `miden-prover` to use `v0.8.0`
removed `miden-test-utils`